arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1050,7 +1067,8 @@
       "properties": {
         "sasTtlAsIso8601": {
           "description": "SAS time to live. Range: 1 Min - 1 Day.",
-          "type": "string"
```

Comment:
yeah just realized that this should not be date-time at all for most of the properties tagged with iso8601. What is being implied is a duration as defined in iso8601. Look at the "Durations" section here https://en.wikipedia.org/wiki/ISO_8601
I will remove date-time tag and provide an example in the description.